cli/cephadm/cephadm.py

```diff
@@ -105,3 +105,17 @@ def logs(self, fsid, name):
         if isinstance(out, tuple):
             return out[0].strip()
         return out
+
+    def ceph_osd(self, id, conf, keyring):
```

Comment:
one doubt @adityaramteke here, so are these the only params to ceph-osd command?
I mean there should be more isn't it? In that case, can we have these as **args? or similar logic?
Just a thought, what do you think?

Reply:
@pranavprakash20 Yes, that should work, ID is mandatory parameter and rest are additional paramters, I have made the changes accordingly.